schema/F3X.json

```diff
@@ -0,0 +1,1022 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/mjtravers/fecfile-online-sandbox/blob/main/validator/schema/F3X.json",
+    "title": "FEC F3X",
+    "description": "FORM 3X - REPORT OF RECEIPTS AND DISBURSEMENTS FOR OTHER THAN AN AUTHORIZED COMMITTEE",
+    "type": "object",
+    "properties": {
+        "FORM_TYPE": {
```

Comment:
Just a thought, it might be helpful to have a FEC electronic filing format version (in this case, 8.3.0.1 per https://docs.google.com/spreadsheets/d/1ROu6m_9YstuHU6TOxZijdZvNwwbkJknD/edit?rtpof=true) in the schema. I'm not sure there's a higher level than F3X but I believe the version number applies to all FEC forms (F3X, F3, F3P, etc)

Reply:
I like the idea of having a fecfile format version.  Because the transaction formats are coming from a [different excel ](https://docs.google.com/spreadsheets/d/1jZLQWVvXgzpST4bP7tn3SlovUf4pC16e/edit#gid=1384755145) would they have a different version?  There's no semver for this document. would the version be the date?

Reply:
That's a great point @toddlees. I think for now we could use a placeholder - maybe a major version change because it's a breaking change. 9.0.0.0? We should probably move to a semantic versioning system at some point.